scripts/install/setup.sh

```diff
@@ -27,6 +29,27 @@ if [ $NUM_ENABLED_APIS != $NUM_REQUIRED_APIS ]; then
   gcloud services --project $PROJECT_ID enable $REQUIRED_APIS
 fi
 
+bold "Checking for exisiting cluster $GKE_CLUSTER"
+
+CLUSTER_EXISTS=$(gcloud beta container clusters list --project $PROJECT_ID \
+  --filter="name=$GKE_CLUSTER" \
+  --format="value(name)")
+
+if [ -n "CLUSTER_EXISTS" ]; then
+  bold "Retrieving credentials for GKE cluster $GKE_CLUSTER..."
```

Comment:
Didn't run this yet, but looks like these 2 commands will be duplicated in the case where the cluster already exists. Maybe the pair down below (https://github.com/GoogleCloudPlatform/spinnaker-for-gcp/pull/48/files#diff-fa0b116bf7afdf3183c66fc88476db50R163) should be in the `if` block just above?